### ISSUE TITLE
Torch numpy ops tracker

### ISSUE THREAD

**Issue by nkovela1** [keras-team/keras-core#188]
Most Torch numpy ops have been successfully completed. However, we need full implementations of these ops to be functionally equivalent to the numpy ops.

- [x] (@nkovela1) `torch.full` and `torch.full_like` --> implement support for an array being passed in `fill_value` arg, maybe use `np.tile`?
- [x] (@nkovela1) `torch.linspace` and `torch.logspace` --> implement support for arrays being passed in `start` and `stop` args
- [x] (@nkovela1 ) `torch.take` --> resolve the incompatibility in dimensions between the result of `np.take` and `torch.take`
- [x] (@nkovela1) `torch.pad` --> implement padding with non-constant padding for the `symmetric` and `reflect` modes
- [x] (@nkovela1) `torch.split` --> resolve the arg divergence between `np.split` and `torch.split` that produces different results
- [x] (@nkovela1) `torch.trace` --> implement `torch.trace` support for arguments `offset`, `axis1`, `axis2`
- [x] (@nkovela1) `torch.eye` --> implement support for `k` diagonal arg that exists in `np.eye` but not `torch.eye`
- [x] (@nkovela1) `torch.tri` --> create an implementation for scratch based on the logic of `np.tri`, as the `torch.tri` API does not exist

To start implementing these:
1. Add your name to the task. 
2. Remove the corresponding pytest skipif decorator in `numpy_test.py`. This will let you see the error to debug.
3. After implementing, if applicable, merge the test with the part of the numpy test without torch.
4. Delete any corresponding NotImplementedError, if it exists.

Thank you and happy hacking!
